### ISSUE TITLE
feature request: edit suggestion url from :open

### ISSUE THREAD

**Issue by boriselec** [tridactyl/tridactyl#2337]
In vimperator it was allowed to edit suggestion url.
:open -> Tab -> (console looks like :open 'url')

**Comment by bovine3dom** [comment]
We have this sort of already: just press space with a result selected

**Comment by boriselec** [comment]
Thanks! Good to know

**Comment by noctuid** [comment]
It would be nice if there was a setting to have tab automatically insert without the space.

**Comment by bovine3dom** [comment]
Yeah, it would : ). I think we have an issue open for that somewhere. It'd require a fairly big rethink of hours completions work - at the moment they are just based on the text in the command line. We'd also have to think about auto selection, i.e. buffer completions.

**Comment by noctuid** [comment]
>  I think we have an issue open for that somewhere

Found at least one other issue: #420

> https://github.com/tridactyl/tridactyl/issues/2337

Maybe it would be good enough to have a setting to insert the currently highlighted url after typing *anything* other than tab/shift-tab. For example if `https://github.com/tridactyl` was highlighted, you could type `/issues` to go from `:tabopen tri` to `:tabopen https://github.com/tridactyl/issues`. Similarly, it would be nice if `text.backward_kill_word` could insert the url first. Doing this wouldn't require any changes to how completion works.

**Comment by glacambre** [comment]
This is already possible, if you're the kind of person who enjoys 
suffering. I believe `bind --mode=ex a composite text.insert_completion 
; text.insert_text a`, `bind --mode=ex b composite 
text.insert_completion ; text.insert_text b` and so on and so forth for 
every character should do just that, at the cost of making typing 
unbearably slow (and maybe the chars will appear in the wrong order, or 
has this bug been fixed?).

**Comment by bovine3dom** [comment]
An easy thing would be to have a bind that inserts the completion after the excmd and before the "query", e.g. shift space. I'm not totally convinced it would be useful, though :)

**Comment by bovine3dom** [comment]
duplicate of #420
